packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -90,7 +97,7 @@ function NavigableToolbar( { children, focusOnMount, ...props } ) {
 	if ( isAccessibleToolbar ) {
 		return (
 			<Toolbar
-				__experimentalAccessibilityLabel={ props[ 'aria-label' ] }
+				accessibilityLabel={ props[ 'aria-label' ] }
```

Comment:
Do you think this could be just "label"? what's the reasoning for using "accessibilityLabel"?

Reply:
This is so it matches the same name as the [React Native prop](https://reactnative.dev/docs/accessibility#accessibilitylabel), but I have no strong opinions on this. I'm totally open for naming it `label` as this is already used by other components like `Button`. cc @gziolo

Reply:
Unfortunately, @gziolo is away for a few months :P. We'll have to make the decision ourselves. My comment was actually driven by the fact that we already use "label" in other places.

Reply:
Ah, let's go with `label` then. It's better to be consistent with other components.